alonet/detr/production/index_to_name.json

```diff
@@ -0,0 +1,94 @@
+{
+    "0": "__background__",
```

Comment:
Also, for the production is seem to me that the index_to_name must be stored into the model weights using register_buffer so that the serving could work with any DETR/Deformable model. 

Thus, when exporting the model for serving, one must register the index_to_name on the model before to export it. Or, the other solution could be to store theses somewhere else, only when the model is export for serving.

In all cases, this file must not be stored here.

Reply:
Done, filename chages to `index_to_name_things`.json